sys/psa_crypto/psa_persistent_storage/psa_crypto_persistent_storage.c

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2023 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_psa_crypto
+ * @{
+ *
+ * @file
+ * @brief       Implementation of persistent storage for PSA Crypto keys
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <fcntl.h>
+#include "psa/crypto.h"
+#include "psa_crypto_slot_management.h"
+#include "board.h"
+#include "vfs.h"
+#include "fs/littlefs2_fs.h"
+#include "mtd.h"
+
+#define ENABLE_DEBUG    0
+#include "debug.h"
+
+XFA_USE(vfs_mount_t, vfs_mountpoints_xfa);
+
+#define UINT32_T_MAX_STRING_LEN     (10)
+
+psa_status_t psa_write_cbor_encoded_key_slot_to_file(psa_key_id_t id,
```

Comment:
The functions in this file do not care about the data format of the keys (CBOR). Maybe it makes sense to rename them accordingly? Something like `psa_write_encoded_key_to_file`?